pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -296,7 +296,7 @@ func timeLeft(j *batch.Job, since *time.Time) (*time.Duration, *time.Time, error
 		klog.Warningf("Warning: Found Job %s/%s finished in the future. This is likely due to time skew in the cluster. Job cleanup will be deferred.", j.Namespace, j.Name)
 	}
 	remaining := expireAt.Sub(*since)
-	klog.V(4).Infof("Found Job %s/%s finished at %v, remaining TTL %v since %v, TTL will expire at %v", j.Namespace, j.Name, finishAt.UTC(), remaining, since.UTC(), expireAt.UTC())
+	klog.V(4).InfoS("Found Job finished", "jobNamespace", j.Namespace, "job", j.Name, "finishTime", finishAt.UTC(), "remainingTTL", remaining, "startTime", since.UTC(), "expireTime", expireAt.UTC())
```

Comment:
Same as above

Reply:
This can actually use `klog.KRef`, thanks for catching it.